synapse_data_warehouse/synapse/dynamic_tables/V2.40.0__acl_latest_refactored.sql

```diff
@@ -0,0 +1,45 @@
+USE SCHEMA {{database_name}}.synapse; --noqa: JJ01,PRS,TMP
+
+
+CREATE OR REPLACE DYNAMIC TABLE ACL_LATEST
+    TARGET_LAG = '1 day'
+    WAREHOUSE = compute_xsmall
+    AS 
+    WITH dedup_acl AS (
+        SELECT
+            *,
+            parse_json(resource_access) as acl,
+        FROM {{database_name}}.SYNAPSE_RAW.ACLSNAPSHOTS --noqa: TMP
+        WHERE
+            SNAPSHOT_DATE >= CURRENT_TIMESTAMP - INTERVAL '14 days'
+        AND
+            RESOURCE_ACCESS != '[]'
```

Comment:
Based on the Slack thread [here](https://sagebionetworks.slack.com/archives/C05PBT2FUHF/p1739394419004849). I can confirm that are no more `NULL` values in the `acl_latest` table with this filter:

<img width="362" alt="image" src="https://github.com/user-attachments/assets/afe26060-9549-4b7a-b54a-c06c14aec430" />

Before there were:

<img width="297" alt="image" src="https://github.com/user-attachments/assets/c4a1092d-516f-4978-bc6e-2be9de5d66c0" />
.